File: csharp/Chapter07/Department.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace com.db4o.dg2db4o.chapter7
{
    class Department
    {
        string _name;
        Manager _manager;

        public Department(string name, Manager manager)
        {
            _name = name;
            _manager = manager;
        }

        public string Name
        {
            get
            {
                return _name;
            }
            set
            {
                _name = value;
            }
        }

        public Manager Manager
        {
            get
            {
                return _manager;
            }
            set
            {
                _manager = value;
            }
        }

        public override string ToString()
        {
            return Name + " (Department)";
        }
    }
}

File: java/src/com/db4o/dg2db4o/chapter7/Department.java

package com.db4o.dg2db4o.chapter7;

public class Department {
    
    String _name;
    Manager _manager;
    
    public Department(String name, Manager manager) {
        _name = name;
        _manager = manager;
    }
    
    public String getName() {
        return _name;
    }
    public void setName(String value) {
        _name = value;
    }
    
    public Manager getManager() {
        
        return _manager;
    }
    public void setManager(Manager value) {
        _manager = value;
    }
    
    public String toString() {
        return _name + " (Department)";
    }
    
}
